src/shared/components/VideoPreviewBase/VideoPreview.stories.tsx

```diff
@@ -1,25 +1,63 @@
-import React from 'react'
+import React, { useState } from 'react'
 import VideoPreviewBase, { VideoPreviewBaseProps } from './VideoPreviewBase'
 import { Meta, Story } from '@storybook/react'
 import styled from '@emotion/styled'
 import { BrowserRouter } from 'react-router-dom'
+import { css } from '@emotion/react'
+import { OverlayManagerProvider } from '@/hooks'
 
 export default {
   title: 'Shared/VideoPreview',
   component: VideoPreviewBase,
   argTypes: {
     createdAt: { control: 'date' },
     progress: { control: { type: 'range', min: 0, max: 100, step: 1 } },
+    channelAvatarUrl: { table: { disable: true } },
+    thumbnailUrl: { table: { disable: true } },
```

Comment:
Why are we disabling those? I see how it doesn't make sense for the drafts for example, but it'd still be beneficial to be able to check different thumbnails on a regular preview

Reply:
makes sense